etl/steps/data/grapher/ggdc/2020-10-01/ggdc_maddison.meta.yml

```diff
@@ -0,0 +1,67 @@
+definitions:
```

Comment:
@Marigold Should we add origins for this one or is it ok that it works with sources for now?

Reply:
This one already uses the origin propagated from the snapshot.